backend/geonature/utils/config_schema.py

```diff
@@ -485,15 +485,17 @@ class MapConfig(Schema):
                 "code": "limitesadministratives",
                 "label": "Limites administratives (IGN)",
                 "type": "wms",
-                "url": "https://wxs.ign.fr/essentiels/geoportail/r/wms",
+                "url": "https://data.geopf.fr/wms-r",
                 "activate": False,
                 "params": {
-                    "VERSION": "1.3.0",
+                    "sevice": "wms",
```

Comment:
Error : "sevice" instead of "service"

Reply:
J'ai manifestement transféré les modifs d'une PR à l'autre un peu trop rapidement. 
Je corrige, test, et réactive la PR dès que c'est fait.